### PR TITLE
Created base version of files

### PR DESCRIPTION
**Pull request by Hrishikesh-Kalyanaraman**
I've changed logpage and create_csv. On logpage, I've added some extra comments and fixed an issue in the version number (version number was showing as one higher than the actual version number). Also fixed some syntax issues on create_csv.py

### CODE REVIEW

**Comment by zachetienne** [comment]
Hi @Hrishikesh-Kalyanaraman . I was browsing the commit and found several references changed from github.com/EinsteinToolkit to github.com/[your user ID]. Could you confirm this is correct?

File: create_csv.py [hunk 1]
```diff
@@ -108,8 +108,8 @@ def get_warning_thorns(name):
     list_of_builds.append(int(m.group(1))-1) # "version" is one less than number in file
 list_of_builds.sort()
 
-with open('docs/version.txt', 'w' as verfile:
-    print("\n".join(list_of_builds), file=verfile)
+with open('docs/version.txt', 'w') as verfile:
+    print("\n".join([str(build) for build in list_of_builds]), file=verfile) #print accepts str, not int
```
Comment (rhaas80) [inline create_csv.py:112]: it's not "print" who does not accept `int` but `join`.

File: test_nums.csv [hunk 2]
```diff
@@ -285,32 +285,32 @@ Date,Total available tests,Unrunnable tests,Runnable tests,Total number of thorn
 1660575744.0,347,14,333,258,99,331,193,2,17.361666666666626,504,284
 1660698954.0,347,14,333,258,99,331,193,2,21.64500000000008,509,285
 1660784850.0,347,15,332,258,99,331,193,1,22.8283333333334,506,286
```
Comment (rhaas80) [inline test_nums.csv:287]: There should not be any new tests since all tests are run by GitHub's action runner and not on the personal laptops.

**Review by rhaas80 — CHANGES_REQUESTED** [review]

File: test_nums.csv (hunk 2 above)
Comment (rhaas80) [reply test_nums.csv:287]: You can use `git rebase -i origin/gh-pages` to edit the existing commit (or make a copy of the code files you changed, `git reset --hard origin/gh-pages` to undo you commit and re-commit with only the Python code.).

**Pull request closed without merging**